messages/cache_settings/messages.go

```diff
@@ -3,67 +3,67 @@ package cache_settings
-    CacheSettingsCreateFlagL2CachingEnabled           = "Whether slice edge caching is active or not"
-    CacheSettingsCreateFlagSliceConfigurationRange    = "Cache Settings' slice configuration range"
-    CacheSettingsCreateFlagCdnCacheSettingsMaxTtl     = "CDN cache settings' maximum TTL"
-    CacheSettingsCreateFlagBrowserCacheSettingsMaxTtl = "Browser cache settings' maximum TTL"
-    CacheSettingsCreateFlagAdaptiveDeliveryAction     = "Cache Settings' adaptive delivery action"
+    CacheSettingsCreateFlagL2CachingEnabled           = "Whether L2 caching is active or not"
+    CacheSettingsCreateFlagSliceConfigurationRange    = "Informs slice configuration range"
+    CacheSettingsCreateFlagCdnCacheSettingsMaxTtl     = "Informs CDN Cache Settings configuration maximum TTL"
+    CacheSettingsCreateFlagBrowserCacheSettingsMaxTtl = "Informs Browser Cache Settings configuration maximum TTL" 
+    CacheSettingsCreateFlagAdaptiveDeliveryAction     = "Informs the Cache Settings configuration adaptive delivery action." 
 
     // [ update ]
     CacheSettingsUpdateUsage            = "update [flags]"
-    CacheSettingsUpdateShortDescription = "Modifies a Cache Setting"
-    CacheSettingsUpdateLongDescription  = "Modifies a Cache Setting based on given attributes to be used in edge applications"
-    CacheSettingsUpdateOutputSuccess    = "Updated cache setting with ID %d\n"
+    CacheSettingsUpdateShortDescription = "Updates a Cache Settings configuration"
+    CacheSettingsUpdateLongDescription  = "Updates a Cache Settings configuration based on given attributes to be used in edge applications"
+    CacheSettingsUpdateOutputSuccess    = "Updated a Cache Settings configuration with ID %d\n"
 
     // [ describe ]
     CacheSettingsDescribeUsage               = "describe --application-id <application_id> --cache-settings-id <cache-settings-id> [flags]"
-    CacheSettingsDescribeShortDescription    = "Returns information about a specific cache settings"
-    CacheSettingsDescribeLongDescription     = "Returns information about a specific cache settings, based on a given ID, in details"
-    CacheSettingsDescribeFlagApplicationID   = "Unique identifier for an edge application. The '--application-id' flag is mandatory"
-    CacheSettingsDescribeFlagCacheSettingsID = "Unique identifier for an origin. The '--cache-settings-id' flag is mandatory"
+    CacheSettingsDescribeShortDescription    = "Returns information about a specific Cache Settings configuration"
+    CacheSettingsDescribeLongDescription     = "Returns information about a specific Cache Settings configuration, based on a given ID, in details"
+    CacheSettingsDescribeFlagApplicationID   = "Unique identifier for an edge application. The '--application-id' flag is required"
+    CacheSettingsDescribeFlagCacheSettingsID = "Unique identifier for an origin. The '--cache-settings-id' flag is required"
```

Comment:
"Unique identifier for a Cache Settings configuration"
To be honest, this flag is redundant... we do not nee more than one flag for Cache Settings ID... I guess this was an oversight. For now, lets leave it like that, but in a future PR we edit this, removing redundant flags.

Reply:
agreed